### DISCUSSION TITLE
Where do you place your importable scripts?

### DISCUSSION THREAD

**Discussion by gustavotoyota** [quasarframework/quasar#14744] [General - Components / Directives / etc] [answered]
This is a simple question that has been bugging me. Where do you place the scripts that you import into your code?
I don't see any folder made specifically for importable scripts. My solution until now was to create `src/scripts/` and put all my code in there.

Where do you place your importable code?

**Comment by suleimanas** [comment, marked as answer]
it's up to you, your solution is 100% correct
